Sources/TuistKit/Commands/BuildCommand.swift

```diff
@@ -31,31 +49,31 @@ public struct BuildOptions: ParsableArguments {
 
     @Option(
         name: .shortAndLong,
-        help: "Build on a specific device."
+        help: "[Deprecated] Build on a specific device."
     )
     public var device: String?
 
     @Option(
         name: .long,
-        help: "Build for a specific platform."
+        help: "[Deprecated] Build for a specific platform."
     )
     public var platform: String?
 
     @Option(
         name: .shortAndLong,
-        help: "Build with a specific version of the OS."
+        help: "[Deprecated] Build with a specific version of the OS."
     )
     public var os: String?
 
     @Flag(
         name: .long,
-        help: "When passed, append arch=x86_64 to the 'destination' to run simulator in a Rosetta mode."
+        help: "[Deprecated] When passed, append arch=x86_64 to the 'destination' to run simulator in a Rosetta mode."
     )
     public var rosetta: Bool = false
 
     @Option(
         name: [.long, .customShort("C")],
-        help: "The configuration to be used when building the scheme."
+        help: "[Deprecated] The configuration to be used when building the scheme."
```

Comment:
I'm not sure I would deprecate `configuration`. We use `configuration` to choose which binary targets to downloads and need to have a tighter control of it. You can see how it behaves with `tuist generate --configuration X` and `tuist cache --configuration X`. While `tuist build` doesn't use binary cache right now, it will soon.